src/vhost_user/master_req_handler.rs

```diff
@@ -437,7 +419,7 @@ mod tests {
         });
 
         fs_cache
-            .fs_slave_map(&VhostUserFSSlaveMsg::default(), fd)
+            .fs_slave_map(&VhostUserFSSlaveMsg::default(), &fd)
```

Comment:
Although this test passed on my local environment, this line caused a build error on the CI with the following error:
>  the trait `std::os::unix::io::AsRawFd` is not implemented for `i32`

It seems that rust-vmm-ci is using a compiler older than 1.48.0, where [`AsRawFd` was implemented for `RawFd`](https://doc.rust-lang.org/std/os/unix/io/trait.AsRawFd.html#impl-AsRawFd-13).

@andreeaflorescu Is it possible to upgrade the version of cargo for CI?

Reply:
If we can't upgrade the toolchain we can wrap this with a newtype:

```rust
struct BorrowedFd<'a>(&'a RawFd);

impl<'a> AsRawFd for BorrowedFd<'a> {
    fn as_raw_fd(&self) -> RawFd {
        *self.0
    }
}
```

Reply:
Okay, I defined this wrapper type in the test function